packages/common/src/core/schemas/shared/CatalogSchema.ts

```diff
@@ -128,6 +128,9 @@ export const CollectionSchema: IConfigurationSchema = {
   properties: {
     label: {
       type: "string",
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      format: "isOnlyWhitespace",
```

Comment:
I feel like we'd want the name to be the inverse of this, e.g. "isNotWhitespace" or something like that. This kind of makes it seem like the validation will check that only white space is entered?